master/getting-started/bare-metal/bare-metal.md

```diff
@@ -476,31 +477,73 @@ outside the cluster:
    normal host endpoint policy).  In other words, if a host endpoint is defined
    but has no pre-DNAT policies that explicitly allow or deny a particular
    incoming packet, that packet is allowed to continue on its way, and will
-   then be accepted or dropped according to workload policy (if it is going to
-   a local workload) or to normal host endpoint policy (if not).
+   then be accepted or dropped according to normal host endpoint policy applied on
+   forward traffic or a workload policy.
+
+## Policy on forward traffic
+If `applyOnForward` is `false`, host endpoint policy applies to traffic to/from
+ local process only. 
+
+If `applyOnForward` is `true`, host endpoint policy applies to forward traffic 
+include:
```

Comment:
This sentence is grammatically incorrect. I am not quite sure what you are trying to say here. If there is some type of forward traffic that is not listed, to which the host endpoint policy gets applied, then: "traffic include" --> "traffic, including"

If all the types of forward traffic to which the host endpoint policy gets applied to are listed, then: "to forward traffic include:" --> "to the following forward traffic:"  (delete "include")